projects/ServoControl/config/UOStateMachine.cpp

```diff
@@ -104,39 +116,83 @@ STATE_DEFINE(UOStateMachine, Control, UOSMData)
         // Check if enable bit is set
         bool enabled = eventNbr > 0 && (eventNbr & EVENT_ENABLE_MASK);
 
-        #if USE_OUT1 == 1
+        #if USE_SV01 == 1
+            if (enabled)
+            {
+                // Open SV01 is the first bit is set
+                bool open = (eventNbr & SV01_EVENT_ENABLE_MASK) > 0;
+
+                gpioData.digitalOutputMap.insert({SV01_NAME, open ? SV01_OPEN : SV01_CLOSE});
+                
+                if(open) {
+                    SPDLOG_LOGGER_INFO(logger, "ServoControlSM::Control SV01 OPEN");
+                } else {
+                    SPDLOG_LOGGER_INFO(logger, "ServoControlSM::Control SV01 CLOSE");
+                }    
+            }
```

Comment:
We have a lot of repetition here. Would we want to move over the common part to a function?